cpp/src/io/comp/cpu_unbz2.cpp

```diff
@@ -90,33 +90,23 @@ namespace cudf {
 namespace io {
 
 // Constants for the fast MTF decoder.
-#define MTFA_SIZE 4096
-#define MTFL_SIZE 16
-
-// Header bytes.
-#define BZ_HDR_B 0x42 /* 'B' */
-#define BZ_HDR_Z 0x5a /* 'Z' */
-#define BZ_HDR_h 0x68 /* 'h' */
-#define BZ_HDR_0 0x30 /* '0' */
+constexpr int mtfa_size = 4096;
+constexpr int mtfl_size = 16;
 
```

Comment:
Making these variables rather than macros doesn't really help the problem that they have unclear names. What is an MTF encoder? What is MTFA and MTFL? Please spell them out.

Reply:
This code is based on libbzip2's decompress.c found [here](https://github.com/centricular/bzip2/blob/master/decompress.c). The names of the constants you see in `cpu_unbz2.cpp` directly come from that file and there isn't much information about what they stand for or signify.

One option could be to leave this file as it were before if changing macro to constexpr doesn't add any value. I think that was the original plan that @vuule and I discussed (to exclude this file) but I forgot about it as I was in the 'flow'.